mslib/mscolab/file_manager.py

```diff
@@ -250,6 +256,59 @@ def modify_user(self, user, attribute=None, value=None, action=None):
             db.session.commit()
         return True
 
+    def upload_file(self, file, upload_folder, file_token=None, subfolder=None, identifier=None):
```

Comment:
The signature can be slimmed down: `upload_folder` and `subfolder` can be one argument, `file_token` is not necessary as it can be created inside the method.